api/include/mfxstructures.h

```diff
@@ -1426,7 +1426,7 @@ typedef struct {
     mfxU16  ROIMode;
     mfxU16  reserved1[10];
 
-    struct  {
+    struct roi {
```

Comment:
first:
you don't need several lines:
```
 struct roi {
...
} ROI[256];
```

second:
don't change API please, it is not allowed without changing API version

Reply:
ok